### PR TITLE
Fix #3127: Split utility.parser into html, image and svg

### LINKED ISSUE #3127: Modularize the parser directory of the utility module.
**Is your feature request related to a problem? Please describe.**
The parser directory consists of a lot of files, which makes it more difficult to organize and make into a Bazel package. It will also unblock parts of the Bazel migration with a split into packages that have no blocked dependencies.

**Describe the solution you'd like**
Create subdirectories to group up the files. e.g. image, html, svg.

### PR DESCRIPTION
**Pull request by FareesHussain**
## Explanation

Fixes #3127
#### HTML
- BulletTagHandler
- ConceptCardTagHandler
- CustomBulletSpan
- CustomHtmlContentHandler
- ExplorationHtmlParserEntityType
- HtmlParser
- HtmlParserEntityTypeModule
- ImageTagHandler
- MathTagHandler
#### Image
- BitmapBlurrer
- BitmapBlurTranformation
- GlideImageLoader
- GlideImageLoaderModule
- ImageAssetFetcher
- ImageLoader
- ImageParsingAnnotations
- ImageParsingModule
- ImageTargets
- RespositoryGlideModule
- RespositoryModelLoader
- TestGlideImageLoader
- TextPictureDrawable
- UrlImageParser
#### SVG
- BlockPictureDrawable
- BlockSvgDrawableTranscoder
- ScalableVectorGraphic
- SvgBlurTransformation
- SvgDecoder
- SvgPictureDrawable
- TestSvgDrawableTranscoder

 
## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

File: .idea/runConfigurations.xml [hunk 1]
```diff
@@ -3,10 +3,11 @@
   <component name="RunConfigurationProducerService">
     <option name="ignoredProducers">
       <set>
+        <option value="com.android.tools.idea.compose.preview.runconfiguration.ComposePreviewRunConfigurationProducer" />
```
Comment (fsharpasharp) [inline .idea/runConfigurations.xml]: There's a change here in  .idea

**Review by fsharpasharp — COMMENTED** [review]
The split looks good to me. I think the checks are getting affected by https://github.com/oppia/oppia-android/issues/3157

File: .idea/runConfigurations.xml (hunk 1 above)
Comment (FareesHussain) [reply .idea/runConfigurations.xml]: I missed this among all other files. Reverted now

**Review by anandwana001 — APPROVED** [review]
LGTM

**Review by rt4914 — APPROVED** [review]
LGTM, Thanks.

File: .idea/runConfigurations.xml (hunk 1 above)
Comment (rt4914) [reply .idea/runConfigurations.xml]: Verified and Resolved

**Review by fsharpasharp — APPROVED** [review]

**Comment by fsharpasharp** [comment]
@vinitamurthi can you take a look at this?

**Review by vinitamurthi — APPROVED** [review]

**Pull request merged**